scripts/data_processing/PPI/ngs_subnational_capacity_data.R

```diff
@@ -79,8 +82,17 @@ NGS.map <- filter(Metadata, submission_date >= today()-365) %>%
     mutate(seq_pc = round(seq_pc, digits = 1), log_seqpc = round(log_seqpc, digits = 2))
 
 #### 3) Export data ####
+#### Get shapefiles for all countries
+## Import shapefile for country borders from repo & save as a csv
+SHAPEFILES_FOR_FLOURISH_PATH <- url('https://raw.githubusercontent.com/PandemicPreventionInstitute/NGS-Capacity-map/main/data/Geospatial_Data/geometric_polygons_country.txt') # shapefiles for mapping
+shapefile <- read_delim(SHAPEFILES_FOR_FLOURISH_PATH, delim = "\t", show_col_types = FALSE) 
+
+## Before exporting filter out countries in the NGS map data
+filter(shapefile, !c(code %in% unique(NGS.map$iso3))) %>% 
```

Comment:
Hey, so I think the point of this was actually to not filter out the countries not in NGS data (so it doesn't warp the map). Those ones would just have NAs!